ftw/ruleset.py

```diff
@@ -24,22 +25,26 @@ def __init__(self, output_dict):
             )
         self.output_dict = output_dict
         self.status = int(output_dict[self.STATUS]) \
-            if self.STATUS in output_dict else None
-        self.html_contains_str = self.process_regex(self.HTML)
-        
+            if self.STATUS in self.output_dict else None
+        self.response_contains_str = self.process_regex(self.RESPONSE)
         self.no_log_contains_str = self.process_regex(self.NOTLOG)
         self.log_contains_str = self.process_regex(self.LOG)
-        if self.status is None and self.html_contains_str is None \
+        self.expect_error = bool(self.output_dict[self.ERROR]) if \
+            self.ERROR in self.output_dict and \
+            isinstance(self.output_dict[self.ERROR], bool) else None
```

Comment:
dont need to use "isinstance" here. Python uses duck typing, so no need for you to check if it is an instance